test/index.js

```diff
@@ -255,15 +255,21 @@ testCommand ('bin/doctest --print test/commonjs/exports/index.js', {
     '__doctest.enqueue({',
     '  type: "input",',
     '  thunk: function() {',
-    '    return exports.identity(42);',
+    '    return (',
+    '      exports.identity(42)',
+    '    );',
     '  }',
     '});',
     '__doctest.enqueue({',
     '  type: "output",',
     '  ":": 2,',
     '  "!": false,',
     '  thunk: function() {',
-    '    return 42;',
+    '    return [',
+    '      {channel: null, value: (',
+    '        42',
+    '      )}',
```

Comment:
This is fine, although I would prefer one of the following styles:

```javascript
{channel: null, value: (42)}
```

```javascript
{
  channel: null,
  value: (
    42
  )
}
```

Reply:
The reason for the newlines is to allow a user to inject `42 // Expect forty-two` there. Without the newlines that would cause the closing bracket to be commented out.

Reply:
I see. Could we use the second of my preferred styles?

Reply:
Ideally we'd use an immediately invoked implicitly returned arrow function, because it would allow for semicolon injection as well as comments.

Reply:
I pushed a commit which changes the formatting. :)

Reply:
:+1: